internal/command.go

```diff
@@ -131,6 +131,7 @@ func NewConfluentCommand(cfg *config.Config) *cobra.Command {
 	cmd.AddCommand(streamshare.New(prerunner))
 	cmd.AddCommand(update.New(cfg, prerunner))
 	cmd.AddCommand(version.New(prerunner, cfg.Version))
+	cmd.AddCommand(ccl.New(cfg, prerunner))
```

Comment:
Please alphabetize this list.

Otherwise, lgtm

Reply:
move it under `connect` for `custom-code-logging`